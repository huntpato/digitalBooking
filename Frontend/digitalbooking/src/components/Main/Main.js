import React from "react";
import { SearchBar } from "../SearchBar/SearchBar";
import { CategoryContainer } from "../CategoryContainer/CategoryContainer";
import { Recommendations } from "../Recommendations/Recommendations";
import styles from "./Main.module.css";

export const Main = () => {
  const { main } = styles;


  return (
    <main className={main}>
      <SearchBar />
      <CategoryContainer />
      <Recommendations />
    </main>
  );
};
