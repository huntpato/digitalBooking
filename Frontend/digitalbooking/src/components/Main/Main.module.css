main {
  -webkit-animation: fade-in 0.3s cubic-bezier(0.39, 0.575, 0.565, 1) both;
  animation: fade-in 0.3s cubic-bezier(0.39, 0.575, 0.565, 1) both;
  min-height: 93vh;
  padding-top: 80px;
}

/*@media (max-width: 480px){
    main{
        background-color: yellow;
    }
}*/

@-webkit-keyframes fade-in {
  0% {
    opacity: 0;
  }
  100% {
    opacity: 1;
  }
}
@keyframes fade-in {
  0% {
    opacity: 0;
  }
  100% {
    opacity: 1;
  }
}
