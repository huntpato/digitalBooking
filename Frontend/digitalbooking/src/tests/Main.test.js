import React from 'react';
import { shallow } from 'enzyme';
import { Main } from '../components/Main/Main'
import '@testing-library/jest-dom'
import { expect } from '@jest/globals';

describe('Pruebas en componente <Main/>', ()=> {

    const wrapper= shallow(<Main />);

    test('Comprobar que <Main/> renderiza correctamente.', () => {
    
        expect( wrapper ).toMatchSnapshot();
      
    })

    test('Verificar que el componente carga la clase de CSS correctamente', () => {

        const main = wrapper.find('main');
        expect(main.hasClass('main')).toBe(true);
    })
    

})
