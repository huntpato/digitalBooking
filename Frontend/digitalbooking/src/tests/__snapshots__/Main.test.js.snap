// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en componente <Main/> Comprobar que <Main/> renderiza correctamente. 1`] = `
<main
  className="main"
>
  <SearchBar />
  <CategoryContainer />
  <Recommendations />
</main>
`;
